src/components/shared/Entity/hooks/useEntityWorkflowHelpers.ts

```diff
@@ -142,21 +143,30 @@ function useEntityWorkflowHelpers() {
                 route = customRoute;
             }
 
+            logRocketConsole('EntityWorkflow:exit:navigate');
             navigate(route, { replace: true });
         },
         [catalogName, generatePath, navigate, resetState]
     );
 
     // Form Event Handlers
-    const closeLogs = useCallback(() => {
-        setFormState({
-            showLogs: false,
-        });
+    const closeLogs = useCallback(
```

Comment:
This function will need to be explicitly called wherever it is used (i.e., `LogDialogActions`, `HeaderLogs` [unused component]) since an input argument has been introduced. Otherwise, `navigateOnly` will be a button event object and, thus, always truthy.

Reply:
dang it... yeah... I'll just take this out as it was a quick add at the end to try to make things better

Reply:
also... this was _really_ dumb since we already return `exit`